engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicStateStatus.scala

```diff
@@ -0,0 +1,61 @@
+package pl.touk.nussknacker.engine.management.periodic
+
+import pl.touk.nussknacker.engine.api.deployment.ProcessActionType.ProcessActionType
+import pl.touk.nussknacker.engine.api.deployment.StateStatus.StatusName
+import pl.touk.nussknacker.engine.api.deployment.{CustomStateStatus, FailedStateStatus, ProcessActionType, RunningStateStatus, StateDefinitionDetails, StateStatus}
+
+import java.net.URI
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+object PeriodicStateStatus {
+
+  //without seconds because we do not deploy with that precision
+  private val format = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
```

Comment:
Let's move this val to RichLocalDateTime

Reply:
I'd rather make this public like most of constants and leave on this level. To be able to use `PeriodicStateStatus.format` in other places like tests

Reply:
If it's constant, it should start with a big letter -> Format?

Reply:
Made this val public.